### PR TITLE
Updated build tools and sample to use AppCompat

### PR DESCRIPTION
**Pull request by emanuelet**

### CODE REVIEW

**Comment by jacobtabak** [comment]
thanks for this! can you please make the indentation consistent with the rest of the project? (2 spaces)

File: build.gradle [hunk 1]
```diff
@@ -19,8 +19,9 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 22
-    buildToolsVersion = '22.0.1'
+    compileSdkVersion = 23
+    buildToolsVersion = '23.0.1'
```
Comment (jaredsburrows) [inline build.gradle]: Build tools 23.0.2 is out.

File: sample/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="android:Theme.Holo.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```
Comment (jaredsburrows) [inline sample/src/main/res/values/styles.xml]: Thank you.

File: build.gradle [hunk 3]
```diff
@@ -19,8 +19,9 @@ allprojects {
...
+    supportLibsVersion = '23.1.1'
     targetSdkVersion = 22
     minSdkVersion = 14
```
Comment (jaredsburrows) [inline build.gradle]: The minSDK can now be API 7.

**Comment by jaredsburrows** [comment]
@emanuelet Can you make those changes really fast?

@jacobtabak After that, can you merge this and republish?

**Comment by emanuelet** [comment]
@jaredsburrows done

**Comment by jaredsburrows** [comment]
@emanuelet Thank you for the fast changes. I ran `gradlew lintDebug` and didn't find any compatibility issues.

@jacobtabak Can we get this in here?

**Comment by jacobtabak** [comment]
I'll merge and make a release today - by the way @emanuelet, if you add more commits to a pull request please comment again, otherwise the repo owner doesn't get notified by default!

**Comment by jacobtabak** [comment]
We can't set min SDK to 7 because of #25. That being the case, I'm not sure there's anything in this PR that would warrant a release. Let me know if you think otherwise.

**Comment by jaredsburrows** [comment]
@jacobtabak Can you at least lower it to 9, 10 or 11?

**Comment by jacobtabak** [comment]
@jaredsburrows it doesn't work on API9. have you tried it?

**Comment by jaredsburrows** [comment]
@jacobtabak I have not but lint didn't bring up an issues. Can we atleast keep it as low as API 11?

**Comment by jacobtabak** [comment]
Sure, I know it doesn't work on 9 but it should be fine on 11.

**Comment by jaredsburrows** [comment]
@jacobtabak Sounds good. After you merge can you make a release?

**Comment by emanuelet** [comment]
@jacobtabak @jaredsburrows done

**Comment by jaredsburrows** [comment]
@emanuelet Thank you

@jacobtabak Should be good to go

**Comment by jaredsburrows** [comment]
@jacobtabak Can we merge and release?

**Comment by jacobtabak** [comment]
@emanuelet thanks again! I will manually merge (there are some remaining style inconsistencies). 

@jaredsburrows personalized releases cost $10 https://cash.me/$JacobTabak/10

**Comment by JakeWharton** [comment]
Can we set minSdk to 15?

**Comment by jaredsburrows** [comment]
@emanuelet Can you list them so we can get them fixed?

@jacobtabak This is an open source project. Why would I pay for something I can just configure myself, @emanuelet and I both made suggestions on this pull.

@JakeWharton Why? What about apps that still support API 11+?

**Comment by jaredsburrows** [comment]
@emanuelet @jacobtabak Actually, after running `gradlew lintDebug`, you will find:
```
../../src/main/res/drawable/white_touch.xml:5: @android:color/holo_blue_light requires API level 14 (current min is 11)

   2 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   3     <item android:state_pressed="true">
   4         <shape android:shape="rectangle">
   5             <solid android:color="@android:color/holo_blue_light" />
   6         </shape>
   7     </item>
```

@android:color/holo_blue_light is API 14.

**Comment by JakeWharton** [comment]
Those apps are stuck in the past and needlessly harming libraries with
requests to lower API levels. There are no devices running 11-14.

**Comment by jaredsburrows** [comment]
@jacobtabak @JakeWharton If you are really going to bump to 14/15, then we should make sure we are using `AppCompat` themes only for compatibility and not any `Holo` themes.

**Comment by jacobtabak** [comment]
@jaredsburrows to be clear, the min SDK was 14 before you suggested that @emanuelet lower it to 7. It was at 14 because there were issues on API9 and for the reasons that Jake stated above. I'm taking your word for it that it works at API 11.

You hijacked this PR and started demanding for a release because you (and only you) wanted to lower the min SDK level.

> Why would I pay for something I can just configure myself

That's a really great question. I would answer that with another question: Why would you demand that I spend my time making a release just for you if you can just configure it yourself?

That said, I made a release.  Enjoy.

Two pieces of advice for the future:

1. If you want to make a change in an open source project (like changing the minSDK), make a separate PR for it and we can discuss it there. 
2. Never pester an open source maintainer to make a release just for you. We hate it.

**Comment by jaredsburrows** [comment]
@jacobtabak **You are overreacting over the Internet**. This is your code. 

@emanuelet was kind enough to submit a pull request to help keep this repository alive. I saw he was adding `AppCompat` so I made a suggestion to lower `minSdk` because AppCompat will take care of the compatibility issues. With that said, you **do not have to accept his pull request** or **my suggestions**. **(This is your choice)**

@JakeWharton was suggesting the `minSdk` of 15 because most devices use APIs 15+. **(And again, this is your choice)**

@jacobtabak **After all this, please do not get upset and "offer advice".** If you do not want to change the `minSdk`, simply say **"I do not want to lower it"**. If you want to keep the `minSdk` at 14, then keep it. I just rather use `AppCompat` instead of `Holo` themes. **(You own this repository, you have the final say)**

**Comment by jacobtabak** [comment]
I'm not going to argue with you - I just want to make sure you understand the difference between "getting annoyed" and "getting upset". You asked three times in this thread for a release, presumably because you want to use this library in an app with MinSDK < 14. 

Please, for the sanity of other open source maintainers, just take the advice.

**Comment by jaredsburrows** [comment]
@jacobtabak I want to thank you for merging @emanuelet's pull request. I **did not need** for you to change the `minSdk` to use it in an app if I chose to use it, I can simply override it in the manifest.

**Comment by JakeWharton** [comment]
Stahp

**Comment by emanuelet** [comment]
relax guys

**Pull request merged**